test/e2e/test_worker_status.py

```diff
@@ -130,3 +103,92 @@ def check_worker_processes_exist() -> None:
 
         # Check that there are worker processes running
         check_worker_processes_exist()
+
+    @pytest.mark.skipif(
+        os.environ["OPERATING_SYSTEM"] == "linux",
```

Comment:
This would fail if we were compiling on a MAC?

Reply:
We only have Linux and Windows worker agents.